todoman/cli.py

```diff
@@ -282,21 +307,19 @@ def edit(ctx, id, todo_properties, interactive):
 @cli.command()
 @pass_ctx
 @with_id_arg
+@catch_errors
```

Comment:
I don't know how to do this. I think it would require a subclass of `click.Group